augur/argparse_.py

```diff
@@ -14,3 +15,35 @@ def run(args):
 
     parser.set_defaults(__command__ = default_command)
 
+
+def add_command_subparsers(subparsers, commands):
+    """
+    Add subparsers for each command module.
+
+    Parameters
+    ----------
+    subparsers: argparse.ArgumentParser
+        The subparsers created by the parent parser via `parser.add_subparsers()`
```

Comment:
Ah, after doing some more in-depth reading of argparse docs and source code, this should probably be changed:

```suggestion
    subparsers: argparse._SubParsersAction
        The special subparsers action object created by the parent parser via `parser.add_subparsers()`
```

Reply:
Updated in 7e4b216